lib/carbon/conf.py

```diff
@@ -76,7 +76,7 @@
   AMQP_VERBOSE=False,
   AMQP_SPEC=None,
   BIND_PATTERNS=['#'],
-  GRAPHITE_URL='http://127.0.0.1:8000',
+  GRAPHITE_URL='http://127.0.0.1:80',
```

Comment:
The default command line to start a dev server binds on 8080, wouldn't that make more sense ?

Reply:
Good question. 8080 is also good. @DanCech ?

Reply:
Not sure what the best solution is, since in most production deployments it's going to be at 80

Reply:
I would say let's aim for prod deployments here...